### PR TITLE
Add Docs about the Platform

### PR DESCRIPTION
**Pull request by dancn**
Just to check the travis build procedure, still some TODO to address before the merge

### CODE REVIEW

File: docs/platform/configuration.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# Configure the SmartSDK Platform Manager
```
Comment (chicco785) [inline docs/platform/configuration.md]: the guided tour is a user perspective, so administration should not be covered

File: docs/platform/usage.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# Use the SmartSDK Platform Manager
+
```
Comment (chicco785) [inline docs/platform/usage.md]: this content as in the agreed toc should go in the `platform/swarmcluster.md` file

**Review by chicco785 — CHANGES_REQUESTED** [review]

File: docs/platform/configuration.md (hunk 1 above)
Comment (dancn) [reply docs/platform/configuration.md]: Ok, will drop.

File: docs/platform/usage.md (hunk 2 above)
Comment (dancn) [reply docs/platform/usage.md]: Ok, will move there!

**Comment by chicco785** [comment]
@dancn can you rebase and be sure to merge with the "local" cluster set-up content?
thx!

**Review by chicco785 — APPROVED** [review]

**Pull request merged**